patches/net/minecraft/world/level/block/ChorusFlowerBlock.java.patch

```diff
@@ -9,6 +9,22 @@
                  boolean flag = false;
                  boolean flag1 = false;
                  BlockState blockstate = p_220981_.getBlockState(p_220982_.below());
+-                if (blockstate.is(Blocks.END_STONE)) {
++                net.neoforged.neoforge.common.util.TriState soilDecision = blockstate.canSustainPlant(p_220981_, p_220982_.below(), Direction.UP, p_220980_);
++                if (!soilDecision.isDefault()) flag = soilDecision.isTrue();
++                else if (blockstate.is(Blocks.END_STONE)) {
```

Comment:
If you move this `else` onto its own line:
```java
else
if (blockstate.is(Blocks.END_STONE)) {
```
Then this hunk won't have to be removing a line